publication-rest/src/main/java/no/unit/nva/publication/create/CreatePublicationHandler.java

```diff
@@ -1,11 +1,13 @@
 package no.unit.nva.publication.create;
 
+import static nva.commons.apigateway.AccessRight.PUBLISH_THESIS;
```

Comment:
PUBLISH_THESIS is bad naming imho. It is not about publishing, but rather creating a publication of certain types, isn't it?

Reply:
I tried to keep it is style with the rest of AccessRights
```
    PUBLISH_FILES,
    PUBLISH_METADATA,
    PUBLISH_THESIS,
    PUBLISH_THESIS_READ_EMBARGOED_FILES,
    APPROVE_DOI_REQUEST,
    REJECT_DOI_REQUEST,
    READ_DOI_REQUEST,
    APPROVE_PUBLISH_REQUEST,
    EDIT_OWN_INSTITUTION_RESOURCES,
    EDIT_OWN_INSTITUTION_USERS,
    EDIT_OWN_INSTITUTION_PROJECTS,
    EDIT_OWN_INSTITUTION_PUBLICATION_WORKFLOW,
```
Feel free to come up with suggestions.

Reply:
EDIT_OWN_INSTITUTION_RESOURCE_THESIS ?

Reply:
Are there users that has access to other institutions resources?